apps/frontend/src/main.tsx

```diff
@@ -74,9 +74,13 @@ const router = createBrowserRouter([
                 path: "/:orgSlug/projects/:projectSlug",
                 element: <AppCore />,
                 children: [
+                    // {
+                    //     path: "settings",
+                    //     element: <Settings />,
+                    // },
                     {
-                        path: "settings",
-                        element: <Settings />,
+                        path: ROUTES.BILLING,
+                        element: <Billing />,
```

Comment:
@afolasope I don't think we need this, the billing screen is just a tab in the workspace settings page.